src/Testcontainers/Configurations/WaitStrategies/ExternalWaitStrategyExtension.cs

```diff
@@ -0,0 +1,24 @@
+namespace DotNet.Testcontainers.Configurations
+{
+  using DotNet.Testcontainers.Builders;
+  using JetBrains.Annotations;
+
+  public static class ExternalWaitStrategyExtension
```

Comment:
Any reason why you choose an extension? I think we can put this into `WaitForContainerOS`.